metrics/configs/flakes-config.yaml

```diff
@@ -105,21 +98,19 @@ jqfilter: |
         (.name): (.flakes|tonumber)}) ])| add
   }})] | add
 
-jqmeasurements: |
-  ([(.[] | .runs|tonumber)] | add) as $totalruns |
-  ([(.[] | .commits|tonumber)] | add) as $totalcommits |
-  [(.[] | {
-    measurement: "flakes",
-    tags: {
-      job: (.job)
-    },
-    fields: {
-      build_consistency: (.build_consistency|tonumber),
-      commit_consistency: (.commit_consistency|tonumber),
-      build_consistency_weighted: ((.build_consistency|tonumber) * (.runs|tonumber) / $totalruns),
-      commit_consistency_weighted: ((.commit_consistency|tonumber) * (.commits|tonumber) / $totalcommits),
-      flakes: (.flakes|tonumber),
-      first_stamp: (.first_stamp),
-      last_stamp: (.last_stamp)
-    }
-  })]
+# No backfilling is used since this metric is only used to display a table with the currently flaky jobs/tests.
+measurements:
+  jq: |
+    [(.[] | {
+      measurement: "flakes",
+      tags: {
+        job: (.job)
+      },
+      fields: {
+        consistency: (.commit_consistency|tonumber),
+        flakes: (.flakes|tonumber),
+        flakiest: (if (.flakiest | has(0)) then (.flakiest[0].flakes + " flakes: " + .flakiest[0].name) else ("") end),
+        flakier: (if (.flakiest | has(1)) then (.flakiest[1].flakes + " flakes: " + .flakiest[1].name) else ("") end),
+        flaky: (if (.flakiest | has(2)) then (.flakiest[2].flakes + " flakes: " + .flakiest[2].name) else ("") end)
```

Comment:
I couldn't figure out how to do this nicely in influxdb, there is no notion of repeated fields so I think my options are:

A) Concatenate the list into a single field (which doesn't display nicely)
or
B) Explicitly define buckets to put the first few list elements in.

I went with B since we only care about the first few flakiest tests anyways, we don't need the full list.